scripts/setup-credentials.ps1

```
@@ -0,0 +1,30 @@
+if ([string]::IsNullOrEmpty($dockerEmail)) { $dockerEmail = "dev@helios.io" }
+
+Write-Host "Creating docker-registry secret..."
+kubectl create secret docker-registry docker-credentials `
+    --docker-server=$dockerServer `
+    --docker-username=$dockerUser `
+    --docker-password=$dockerPass `
+    --docker-email=$dockerEmail `
+    --dry-run=client -o yaml | kubectl apply -f -
+
+Write-Host "Patching pipeline service account..."
+$sa = kubectl get sa pipeline -n default -o name 2>$null
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'sa' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)